src/ServiceWorker.js

```diff
@@ -48,11 +49,15 @@ self.addEventListener('activate', event => {
 
 // Intercept fetch
 self.addEventListener('fetch', event => {
-    // forward request
-    // @ts-ignore Property 'respondWith' does not exist on type 'Event'.ts,
-    // Property 'request' does not exist on type 'Event'.ts
-    event.respondWith(fetch(event.request, {
-        // omit cookie transmission
-        credentials: 'omit',
-    }));
+    // @ts-ignore Property 'request' does not exist on type 'Event'.ts
+    const requestHost = new URL(event.request.url).host;
+    if (requestHost === location.host) {
```

Comment:
Please add a comment with the link https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy explaining, why we test on host instead of origin.